spec/system/consumer/map_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'system_helper'
+
+RSpec.describe 'Map' do
+  context 'map can load' do
+    it 'does not show alert' do
+      url_whitelist = page.driver.browser.url_whitelist
+      page.driver.browser.url_whitelist = nil
+
+      assert_raises(Capybara::ModalNotFound) do
+        accept_alert { visit '/map' }
+      end
+
+      page.driver.browser.url_whitelist = url_whitelist
+    end
+  end
+
+  context 'map cannot load' do
+    it 'shows alert' do
+      message = accept_alert { visit '/map' }
+      expect(message).to eq(I18n.t('gmap_load_failure'))
```

Comment:
As a general rule we prefer to check with the actual message, so we can check the translation is working ,ie: 
```suggestion
      expect(message).to eq("Unable to load map. Please check your browser settings and allow 3rd party cookies for this website.")
```

Reply:
The string needs to be broken across multiple lines in this case to keep rubocop happy. I pushed a change that uses `String#squish`, but let me know if you have a preferred method for this.

Reply:
I personally prefer to split the string with a `\` but `squish` work as well. ie:
```ruby
 expect(message).to eq(
  "Unable to load map. Please check your browser" \
  " settings and allow 3rd party cookies for this website."
)

Reply:
I like this better. I committed this changed.